src/main/resources/templates/websocket-player.html

```diff
@@ -3,6 +3,7 @@
 <head>
     <meta charset="UTF-8"/>
     <title>Tech Poker</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
The problem with the viewport was that it doesn't fit the entire content in one screen without scrolling, at least on my device (vovo has the same) I can only see up to card 13. 
I didn't spent much time trying to figure out how to make it work with this setting and just left it out.

Reply:
As you have already seen, I removed the viewport again. To be honest, I haven't tested it on my smartphone yet.

Reply:
I have to delete it in the vote template as well for the plain variant. This will come in the next commit.